src/components/Footer/Footer.scss

```
@@ -0,0 +1,403 @@
+// @media (max-width:1200px) {
+.footerContainer {
+  .iconWrapper {
+    .iconList {
+      display: flex;
+      justify-content: space-around;
+      padding: 20px 100px;
+      background: #fff;
+
+      button {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+
+        i {
+          font-size: 17px;
+        }
+
+        span {
+          margin-top: 10px;
+          font-size: 11px;
+        }
+      }
+    }
+  }
+
+  .footerLinkWrapper {
+    display: flex;
+    justify-content: space-between;
+    padding: 25px 60px;
+    border-bottom: 1px solid #e5e5e5;
+    background: #f4f4f4;
+
+
+    .kakaoWrapper {
+
+      .kakaoTitle {
+        display: flex;
+        align-items: center;
+
+        img {
+          width: 33px;
+          height: 31px;
+          overflow: hidden;
+        }
+      }
+
+      h3 {
+        display: flex;
+        margin-left: 2px;
+        font-size: 13px;
+        font-weight: 700;
+      }
+    }
+
+    .kakaoContent {
+      width: 370px;
+      margin: 5px 0;
+      white-space: normal;
+      font-size: 11px;
+    }
+  }
+
+  .kakaoBtn {
+    height: 30px;
+    margin: 5px 0;
+    padding: 5px 17px;
+    border: 1px solid #545454;
+    border-radius: 27px;
+    font-size: 11px;
+    background: #292929;
+    color: #fff;
+
+    &:hover {
+      border: 1px solid #e5e5e5;
+      background: #fff;
+      color: #000;
+    }
+  }
+
+  .totalStore {
+    display: flex;
+    margin: 15px 0;
+    font-size: 9px;
+    font-weight: 700;
+    color: #545454;
+
+  }
+
+  .storeSearchBtn {
+    padding: 7px 160px;
+    border: 1px solid #f4f4f4;
+    border-radius: 27px;
+    background: #fff;
+  }
+
+  .aboutWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+  }
+
+  .collectionWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+
+  }
+
+  .helpWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+  }
```

```suggestion
  h4 { // 혹은 .menuTitle 과 같이 클래스 셀렉터 사용
    margin-bottom: 14px;
    font-size: 9px;
    font-weight: 600;
    color: #545454;
  }
```
동일한 스타일에 대해서 코드가 중복되고 있는 것 같습니다. 특별한 이유가 없다면 동일한 스타일은 동일한 코드로 한번에 표현하는 것이 어떨까요?